### PR TITLE
Add application details page

### PR DESCRIPTION
**Pull request by soupette**
#### Description of what you did:

This PR adds an application details page.

### CODE REVIEW

File: packages/strapi-admin/admin/src/components/LeftMenu/LeftMenuLink/LeftMenuLinkContent.js [hunk 1]
```diff
@@ -67,6 +69,13 @@ const LeftMenuLinkContent = ({ destination, iconName, label, location }) => {
     >
       <LeftMenuIcon icon={iconName} />
       {content}
+      {notificationsCount > 0 && (
+        <NotificationCount>
+          <Text fontWeight="bold" fontSize="xs" lineHeight="14px" color="white">
```
Comment (HichamELBSI) [inline packages/strapi-admin/admin/src/components/LeftMenu/LeftMenuLink/LeftMenuLinkContent.js:74]: The text color is not the right one. It should be `#919BAE`.

File: packages/strapi-admin/admin/src/plugins.js [hunk 2]
```diff
@@ -23,15 +23,15 @@ window.strapi = Object.assign(window.strapi || {}, {
...
+  //   .default,
+  // 'strapi-plugin-content-type-builder': require('../../../strapi-plugin-content-type-builder/admin/src')
+  //   .default,
 
-  'strapi-plugin-email': require('../../../strapi-plugin-email/admin/src').default,
-  'strapi-plugin-upload': require('../../../strapi-plugin-upload/admin/src').default,
-  'strapi-plugin-graphql': require('../../../strapi-plugin-graphql/admin/src').default,
+  // 'strapi-plugin-email': require('../../../strapi-plugin-email/admin/src').default,
+  // 'strapi-plugin-upload': require('../../../strapi-plugin-upload/admin/src').default,
+  // 'strapi-plugin-graphql': require('../../../strapi-plugin-graphql/admin/src').default,
```
Comment (HichamELBSI) [inline packages/strapi-admin/admin/src/plugins.js:36]: I think we should uncomment those lines 😆

File: packages/strapi-admin/admin/src/components/LeftMenu/LeftMenuLink/A.js [hunk 3]
```diff
@@ -10,7 +11,6 @@ const A = styled.a`
   cursor: pointer;
   color: ${props => props.theme.main.colors.leftMenu['link-color']};
   text-decoration: none;
-  display: block;
   -webkit-font-smoothing: antialiased;
 
   &:hover {
```
Comment (HichamELBSI) [inline packages/strapi-admin/admin/src/components/LeftMenu/LeftMenuLink/A.js:16]: I put this comment here but it is not related to this file. 

Something is wrong with the menu link hover background (Screenshot).

The way to fix that : 
1) Go to `/strapi-admin/admin/src/components/LeftMenu/LeftMenuLinkSection/LeftMenuListLink.js`
2) Remove the `margin-right: 28px;`
3) Go to `/strapi-admin/admin/src/components/LeftMenu/LeftMenuLink/A.js`
4) Add `padding-right: 2.3rem;` to the `&.linkActive` line 34.

Capture d’écran 2020-10-15 à 14 48 30

**Review by HichamELBSI — CHANGES_REQUESTED** [review]

File: packages/strapi-admin/admin/src/containers/ApplicationInfosPage/index.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+      <BaselineAlignement top size="3px" />
+      <Bloc>
+        <Padded left right top size="smd">
+          <Padded left right top size="xs">
+            <Flex justifyContent="space-between">
+              <Detail
+                link={upgradeLink}
+                title="STRAPI VERSION"
+                content={`v${appInfos.strapiVersion}`}
+              />
```
[the remark this answers is not in the document]
Comment (soupette) [reply packages/strapi-admin/admin/src/containers/ApplicationInfosPage/index.js:62]: There is a prop called link for this.

File: packages/strapi-admin/admin/src/plugins.js (hunk 2 above)
Comment (soupette) [reply packages/strapi-admin/admin/src/plugins.js:36]: It does not really matter because this file is only used when developing in the monorepo.

**Review by HichamELBSI — APPROVED** [review]
LGTM

**Pull request merged**